htmx-spring-boot/src/main/java/io/github/wimdeblauwe/htmx/spring/boot/mvc/HtmxResponse.java

```diff
@@ -196,96 +249,321 @@ public HtmxResponse and(HtmxResponse otherResponse) {
         mergeMapAndLog(HxTriggerLifecycle.SETTLE, this.triggersAfterSettle, otherResponse.triggersAfterSettle);
         mergeMapAndLog(HxTriggerLifecycle.SWAP, this.triggersAfterSwap, otherResponse.triggersAfterSwap);
 
-        if (otherResponse.getHeaderPushHistory() != null) {
-            this.headerPushHistory = otherResponse.getHeaderPushHistory();
+        if(otherResponse.getPushUrl() != null) {
+            this.pushUrl = otherResponse.getPushUrl();
         }
-        if (otherResponse.getHeaderRedirect() != null) {
-            this.headerRedirect = otherResponse.getHeaderRedirect();
+        if(otherResponse.getRedirect() != null) {
+            this.redirect = otherResponse.getRedirect();
         }
-        if (otherResponse.getHeaderRefresh()) {
-            this.headerRefresh = true;
+        if(otherResponse.isRefresh()) {
+            this.refresh = true;
         }
-        if (otherResponse.getHeaderRetarget() != null) {
-            this.headerRetarget = otherResponse.getHeaderRetarget();
+        if(otherResponse.getRetarget() != null) {
+            this.retarget = otherResponse.getRetarget();
         }
-        if (otherResponse.getHeaderReswap() != null) {
-            this.headerReswap = otherResponse.getHeaderReswap();
+        if(otherResponse.getReswap() != null) {
+            this.reswap = otherResponse.getReswap();
         }
 
         return this;
     }
 
+    @Deprecated
     private boolean same(ModelAndView one, ModelAndView two) {
         if (one == two) {
             return true;
         }
         if (one == null || two == null) {
             return false;
         }
-        if (one.getViewName() != null && one.getViewName().equals(two.getViewName())) {
+        if (one.getViewName() !=null && one.getViewName().equals(two.getViewName())) {
```

Comment:
Please keep the space between `!=` and `null`

Reply:
Done.